src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacNumericValidator.java

```diff
@@ -368,6 +387,16 @@ private double roundStringValueToSigFigs(final String value, final int sigFigs)
 
     }
 
+    /**
+     * Return a double equivalent to value.
+     *
+     * @param value - number, as String, to convert to double
+     * @return the converted number.
+     */
+    private double stringValueToDouble(final String value) {
+        return new BigDecimal(value).doubleValue();
```

Comment:
It's mostly for my benefit when tracing this, and probably not that fantastic, but all the other methods have debug statements right at the start with their names in, like:
```
log.debug("\t[stringValueToDouble]");
```
Could this one have it too for consistency?

Reply:
Added.